pkg/doc/jsonld/context/importer.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package context
+
+import (
+	"bufio"
+	"embed"
+	"encoding/json"
+	"fmt"
+
+	"github.com/piprate/json-gold/ld"
+
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+	"github.com/hyperledger/aries-framework-go/spi/storage"
+)
+
+const (
+	DefaultContextDBName = "ContextsDB"
```

Comment:
Not sure about the "DB" suffix - if we are worried about uniqueness, I suppose we could include jsonld (or similar) in the name.